pssh.py

```diff
@@ -96,7 +96,7 @@ def __init__(self, host,
         :param num_retries: (Optional) Number of retries for connection attempts\
         before the client gives up. Defaults to 3.
         :type num_retries: int
-        :param timeout: (Optional) Number of seconds to timout connection attempts\
+        :param timeout: (Optional) Number of seconds to timeout connection attempts\
         before the client gives up. Defaults to 10.
```

Comment:
Is there a placeholder we can specify here that autofills the default value when docs are generated? Like a `:default:` or something. That would be better than having to update the default value both in the method signature and in the docstring when/if it changes.